main.go

```diff
@@ -84,7 +84,7 @@ func main() {
 			} else {
 				timeout.Stop()
 			}
-			logging.VeryVerbose.Printf("new masters detected: %v", masters)
+			log.Printf("new masters detected: %v", masters)
```

Comment:
Thanks for the PR! It looks like this project does not use `log.Printf` and instead uses `logging.Verbose`, according to [this](https://github.com/mesosphere/mesos-dns/blob/1810c09811dfeb8c9e72ee6c58102313f74e0e1c/logging/logging.go#L13-L24). This seems strange to me, also it looks like in DC/OS mesos-dns is not running in verbose mode by default. You should probably ask @drewkerrigan

Reply:
`logging.Verbose` also requires a flag and I think that having this log line displayed in all cases would be better for users. `log.Printf` is used in `tools/main.go` and `util/util.go`, as `log` was already imported in `main.go` it did not seem illogical to use it but I may be wrong.

Reply:
`logging.Verbose` would be correct here. the philosophy of the `logging` package is that a log message carries an error signal or else it's some kind of an "info" (verbose) message. modules in mesos-dns should use `logging` to generate log messages.

using the stdlib `log` func in main should probably be rewritten. i haven't looked yet at the `util` package.